internal/broadcasters_test.go

```diff
@@ -97,7 +98,7 @@ func TestBroadcasterDataRace(t *testing.T) {
 		func() { b.HasListeners() },
 		func() { b.RemoveListener(nil) },
 	} {
-		const concurrentRoutinesWithSelf = 2
+		const concurrentRoutinesWithSelf = 10
```

Comment:
This change is mainly based on intuition and some experience, but I don't have any actual hard data to back up "more goroutines = higher chance of catching the race".

Reply:
I’m uncertain on best practice there. I was able to reliably reproduce both race conditions with 2, but it should be enough to cause a flaky test at worst. 10 seems fine, I’m just not sure if it adds anything 🤷

Reply:
I'll revert this change.